src/jit/codegenarmarch.cpp

```diff
@@ -3500,11 +3486,20 @@ void CodeGen::genCodeForStoreBlk(GenTreeBlk* blkOp)
 void CodeGen::genScaledAdd(emitAttr attr, regNumber targetReg, regNumber baseReg, regNumber indexReg, int scale)
```

Comment:
This function name and the variable name scale are unfortunate.  Scale implies multiply not shift.